src/coreclr/vm/dllimport.h

```diff
@@ -150,15 +150,15 @@ enum NDirectStubFlags
 #ifdef FEATURE_COMINTEROP
     NDIRECTSTUB_FL_FIELDGETTER              = 0x00002000, // COM->CLR field getter
     NDIRECTSTUB_FL_FIELDSETTER              = 0x00004000, // COM->CLR field setter
-    // unused                               = 0x00008000,
-    // unused                               = 0x00010000,
-    // unused                               = 0x00020000,
+#endif // FEATURE_COMINTEROP
+    NDIRECTSTUB_FL_SUPPRESSGCTRANSITION     = 0x00008000,
+    NDIRECTSTUB_FL_STUB_HAS_THIS            = 0x00010000,
```

Comment:
Are these flags meant to show up in `NDirectStubHashBlob` ? If not, they should be in the internal flags section.